docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -45,6 +45,13 @@ INSERT INTO [ TABLE ] table_identifier [ partition_spec ]
 
     **Syntax:** `PARTITION ( partition_col_name  = partition_col_val [ , ... ] )`
 
+* **column_list**
+
+  An optional parameter that specifies a comma separated list of columns belong to the `table_identifier`.
+  All specified columns should exist in the `table_identifier` and not be duplicated from each other. It includes all columns except the static partition columns.
+  The size of the column list should be exactly the size of the data from `VALUES` clause or query.
+  The order of the column list is alterable and determines how the data from `VALUES` clause or query to be inserted by position.
```

Comment:
How about organizing it like this (it seems some statements above describe the current limitations):
```
  * **colunn_list**
  <param description>
 
  **Note**
  The current behaviour has some limitations:
  1. The column list should contain all the column names in the `table_identifier` table.
  2. ...
```
(ref: https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-qry-select-having.md#L43-L48)